coffee_cmd/src/main.rs

```diff
@@ -98,6 +98,23 @@ async fn main() -> Result<(), CoffeeError> {
             }
             Err(err) => Err(err),
         },
+        CoffeeCommand::Nurse {} => {
+            let mut spinner = term::spinner(&format!("Trying restoring from a corrupted status!"));
+            match coffee.nurse().await {
+                Ok(val) => {
+                    let status = val["nurse"].as_str().unwrap();
+                    if status == "corrupt" {
+                        spinner.message("Storage refreshed");
```

Comment:
Please make a model and not return a plan json here

Reply:
Sure. Just pushed the code with the changes.